app/uk/gov/hmrc/gform/sharedmodel/email/LocalisedEmailTemplateId.scala

```diff
@@ -45,4 +49,6 @@ object LocalisedEmailTemplateId {
     case otherwise =>
       JsError("Invalid email template id definition. Expected json String or json Object, but got: " + otherwise)
   }
+
+  implicit val format: OFormat[LocalisedEmailTemplateId] = OFormatWithTemplateReadFallback(reads)
```

Comment:
Can we unimplicitify `reads` now? Better to have only one implicit read in scope.